.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 
 language: go
 
+# Do not move these lines; they are referred to by README.md.
```

Comment:
tangentially connected to this PR, but, perhaps rename `s/tip/master/` in the line below ?

Reply:
I like history, but sure, that can change at some point.